### DISCUSSION TITLE
Welcome to F* Discussions!

### DISCUSSION THREAD

**Discussion by nikswamy** [FStarLang/FStar#2510] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by aseemr** [comment]
This comment is testing the slack integration for a regular comment.
